### DISCUSSION TITLE
构建自己的杠杆 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#121] [Announcements]
# 构建自己的杠杆 · 构建我的被动收入

终身学习（认知杠杆）* 全栈技术（时间杠杆）* 投资理财（财务杠杆） => 被动收入（人生杠杆）

https://www.bmpi.dev/self/build-leverage/

**Comment by haoweiguang** [comment]
跟naval说的杠杆有点类似
- 人力杠杆
- 资本杠杆就是投资
- 技术杠杆就是复制成本为零的产品

**Reply by madawei2699** [reply]
是的，这么一说倒挺类似的，我刚好也写过他文集的读书笔记：https://www.bmpi.dev/self/how-to-get-rich/
